externs/es6.js

```diff
@@ -394,6 +394,23 @@ function Int8Array(length, opt_byteOffset, opt_length) {}
 /** @type {number} */
 Int8Array.BYTES_PER_ELEMENT;
 
+/** 
+* @param {Array<number>} source
```

Comment:
If there is a good reason to pass null this should be {?Array<number>}. If not then make it {!Array<number>}